packages/rs-dpp/src/data_contract/config/v0/mod.rs

```diff
@@ -89,54 +91,3 @@ impl DataContractConfigV0 {
         Self::default().into()
     }
 }
-
-impl DataContractConfigV0 {
-    /// Retrieve contract configuration properties.
-    ///
-    /// This method takes a BTreeMap representing a contract and retrieves
-    /// the configuration properties based on the values found in the map.
-    ///
-    /// The process of retrieving contract configuration properties is versioned,
-    /// and the version is determined by the platform version parameter.
-    /// If the version is not supported, an error is returned.
-    ///
-    /// # Parameters
-    ///
-    /// * `contract`: BTreeMap representing the contract.
-    /// * `platform_version`: The platform version being used.
-    ///
-    /// # Returns
-    ///
-    /// * `Result<ContractConfig, ProtocolError>`: On success, a ContractConfig.
-    ///   On failure, a ProtocolError.
-    pub(super) fn get_contract_configuration_properties_v0(
```

Comment:
why was this removed?

Reply:
Because we don't use this. It was needed when we had manually defined data contracts with JSON schema. If you need to get a config from value manually you just need to use platform_value::from_value()